src/components/Modals/Nfts/components/NftOverview.tsx

```diff
@@ -9,31 +10,25 @@ import { Row } from 'components/Row/Row'
 import { Text } from 'components/Text'
 import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
 import { markdownLinkToHTML } from 'lib/utils'
-import type {
-  SupportedZapperNetwork,
-  V2NftCollectionType,
-  V2ZapperNft,
-} from 'state/apis/zapper/validators'
-import { ZAPPER_NETWORKS_TO_CHAIN_ID_MAP } from 'state/apis/zapper/validators'
+import type { NftCollectionItem, NftItem } from 'state/apis/nft/types'
 import { selectAssetById } from 'state/slices/assetsSlice/selectors'
 import { useAppSelector } from 'state/store'
 
 type NftOverviewProps = {
-  zapperNft: V2ZapperNft
-  zapperCollection?: V2NftCollectionType[]
+  nftItem: NftItem
+  nftCollection?: NftCollectionItem
 }
 
-export const NftOverview: React.FC<NftOverviewProps> = ({ zapperCollection, zapperNft }) => {
+export const NftOverview: React.FC<NftOverviewProps> = ({ nftItem, nftCollection }) => {
```

Comment:
As previously stated, when the nftApi becomes a bit more normalized and we can make a join between NFTs and collections, we should really only pass the former, which would contain a reference to the collection AssetId